dockers/skylab/pytools/tools/create_loom_slide_seq.py

```diff
@@ -0,0 +1,409 @@
+import argparse
```

Comment:
This script is not used in this version of the pipeline. Not sure if we want to keep it or leave it on the branch.

Reply:
Hi @khajoue2, it looks like this script is in the pytools dir. I am good with keeping it here if you are!